.db.env

```diff
@@ -1,28 +1,25 @@
 # PostgreSQL
-TEST_POSTGRES_BASE_URI="postgres://prisma:prisma@localhost:5432"
 TEST_POSTGRES_URI="postgres://prisma:prisma@localhost:5432/tests"
```

Comment:
```suggestion
# Note that the database name must be `tests` (in our test setup we search and replace that value later)
TEST_POSTGRES_URI="postgres://prisma:prisma@localhost:5432/tests"
```

Reply:
Decided to add a note at top of file instead as otherwise we need to repeat this one too many times all over the files